conformance/utils/echo/pod.go

```diff
@@ -80,22 +80,25 @@ func makeRequest(t *testing.T, exp *http.ExpectedResponse) []string {
 	if exp.Request.Host == "" {
 		exp.Request.Host = "echo"
 	}
-	if exp.Request.Method == "" {
+
+	r := exp.Request
+	protocol := strings.ToLower(r.Protocol)
```

Comment:
do we really need this validation?

I mean, if protocol == grpc, the below line will already be skipped. So if it is empty, it is not grpc. If it is http, it is not grpc :) 

Just giving some thoughts on some simplification here

edit: I can see you just moved the code from above to here now, so if "protocol" is used on other parts of the code feel free to keep it